src/Mesh/babylon.mesh.ts

```diff
@@ -51,6 +51,7 @@
         public _binaryInfo: any;
         private _LODLevels = new Array<Internals.MeshLODLevel>();
         public onLODLevelSelection: (distance: number, mesh: Mesh, selectedLevel: Mesh) => void;
+        public highlight = false;
```

Comment:
No problem. I will find a way to keep my changes without accidently posting them to the main branch. Sorry for the mistake here.